docs/ha-considerations.md

```diff
@@ -131,8 +125,8 @@ defaults
     timeout http-request    10s
     timeout queue           20s
     timeout connect         5s
-    timeout client          20s
-    timeout server          20s
+    timeout client          1800s
+    timeout server          1800s
```

Comment:
what is the significance of increasing these timeouts? can we get away with the shorter ones?